pom.xml

```diff
@@ -44,6 +44,7 @@
         <module>config-model-fat</module>
         <module>config-model</module>
         <module>config</module>
+        <module>configgen</module>
```

Comment:
configgen is built in bootstrap.sh, and it doesn't seem like we include the rest of those modules in the root aggregator pom. If it's necessary or desired, please add a comment.

Reply:
Had to add it to code and run tests in IDE properly. It's not important, I just didn't understand why it was left out. I'll remove it again